### PR TITLE
SUCCESS-1281: Correct IdM code examples

### PR DESCRIPTION
**Pull request by seedifferently**

### CODE REVIEW

File: source/includes/_identitymatch.md [hunk 1]
```diff
@@ -445,19 +447,19 @@ params = {
...
-        'day': 13,
-        'month': 5,
-        'year': 1988
+        "day": 13,
+        "month": 5,
+        "year": 1988
     }
 }
-client.post("/identity/match_config/<algorithm_id>", data=params)
+client.post("/identity/match/<algorithm_id>", data=params)
```
Comment (jeffhoek) [inline source/includes/_identitymatch.md]: does this api path need a `find` suffix here?

File: source/includes/_identitymatch.md [hunk 2]
```diff
@@ -483,7 +485,7 @@ Dictionary<string, object> data = new Dictionary<string, string> {
         { "year", 1988}
     }
 }
-client.request("/identity/match_config/<algorithm_id>", "POST", data)
+client.request("/identity/match/<algorithm_id>", "POST", data)
```
Comment (jeffhoek) [inline source/includes/_identitymatch.md]: same here (missing `find` suffix in path)

File: source/includes/_identitymatch.md [hunk 3]
```diff
@@ -420,18 +422,18 @@ curl -i -H "Authorization: Bearer $ACCESS_TOKEN" -d '{
...
     "birth_date": {
-        'day': 13,
-        'month': 5,
-        'year': 1988
+        "day": 13,
+        "month": 5,
+        "year": 1988
     }
-}' https://platform.pokitdok.com/api/v4/identity/match_config/<algorithm_id>
+}' https://platform.pokitdok.com/api/v4/identity/match/<algorithm_id>
```
Comment (jeffhoek) [inline source/includes/_identitymatch.md]: does this api path need a `find` suffix here?

File: source/includes/_identitymatch.md [hunk 4]
```diff
@@ -356,8 +257,132 @@ let params = [
...
+    "meta": {
+        "...": "..."
+    }
+}
+```
+
+> Example request to retrieve the _canonical_ identity (the representative identity for a set of matching source identity).
+
+```shell
+curl -i -H "Authorization: Bearer $ACCESS_TOKEN" https://platform.pokitdok.com/api/v4/identity/match/test_name_and_strong_id/source/source_id/canonical
```
Comment (mlaldrid) [inline source/includes/_identitymatch.md]: This `test_name_and_strong_id` string is an artifact of some internal example using a dev match configuration. I think that could be distracting in public-facing docs. Maybe  `algorithm_id` for consistency with the rest of the docs would be better.

**Pull request merged**